tests/Auth0Test.php

```diff
@@ -377,7 +390,10 @@ public function testThatGetLoginUrlGeneratesState()
         $parsed_url_query = parse_url( $auth_url, PHP_URL_QUERY );
         $url_query        = explode( '&', $parsed_url_query );
 
+        $this->assertArrayHasKey( 'auth0__webauth_state', $_SESSION );
         $this->assertContains( 'state='.$_SESSION['auth0__webauth_state'], $url_query );
+        $this->assertArrayHasKey( 'auth0__nonce', $_SESSION );
```

Comment:
if this is used for webauth as well, should we keep the consistency of calling it `auth0__webauth_nonce`?

Reply:
Discrepancies with state will be addressed in another PR. Not a big fan of the term "webauth" in this context, might get confused with [WebAuth browser API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API)